dev-requirements.txt

```diff
@@ -16,3 +16,4 @@ Flask>=1.1
 ruff===0.2.2
 # needed for dapr-ext-workflow
 durabletask>=0.1.1a1
+anyio>=4.4.0 # not directly required, pinned by Snyk to avoid a vulnerability
```

Comment:
Pinning dependencies like this causes customers / users to have problems using our SDK together with other third party SDKs. To Avoid conflict we should not require such a high minimum version.

This PR should not be completed.